indico/modules/events/persons/controllers.py

```diff
@@ -313,6 +294,24 @@ def _find_role_members(self, role_ids):
         return itertools.chain.from_iterable(role.members for role in query)
 
 
+class RHEmailEventPersonsPreview(RHEmailEventPersonsBase):
```

Comment:
Why move it around within the file? Makes it harder to review the actual changes... ;)

Reply:
because one of the changes is that it now inherits from a class that was below 😅